inference-engine/tests/unit/cpu/mkldnn_memory_desc_test.cpp

```diff
@@ -45,314 +45,986 @@ TEST(MemDescTest, Conversion) {
-    dnnl::memory::desc tailc_crop_tdesc = blck8_tdesc.submemory_desc(crop_dims, crop_off);
-    ASSERT_FALSE(MKLDNNExtensionUtils::makeDescriptor(tailc_crop_tdesc)->hasLayoutType(LayoutType::nspc));
-}
+    for (const auto &p : payload) {
+        isCompatibleDnnlMemoryDesc(p.first, p.second);
+        isCompatibleDnnlBlockedMemoryDesc(p.first, p.second);
+        isCompatibleCpuBlockedMemoryDesc(p.first, p.second);
+    }
 
-TEST(MemDescTest, constructWithPlainFormat) {
-    GTEST_SKIP();
-}
+    // Should be True after fix
+    auto isCompatibleEquallyStoredInMemory = [] (dnnl::memory::format_tag fmt1, dnnl::memory::format_tag fmt2, const Shape& shape) {
+        DnnlBlockedMemoryDesc blk_desc1(shape, dnnl::memory::data_type::u8, fmt1);
+        MemoryDescPtr cpu_blk_desc_ptr_ncsp = std::make_shared<CpuBlockedMemoryDesc>(blk_desc1.getPrecision(), blk_desc1.getShape(),
+                                                                                     blk_desc1.getBlockDims(), blk_desc1.getOrder(),
+                                                                                     blk_desc1.getOffsetPadding(), blk_desc1.getOffsetPaddingToData(),
+                                                                                     blk_desc1.getStrides());
+
+        DnnlBlockedMemoryDesc blk_desc2(shape, dnnl::memory::data_type::u8, fmt2);
+        MemoryDescPtr cpu_blk_desc_ptr_nspc = std::make_shared<CpuBlockedMemoryDesc>(blk_desc2.getPrecision(), blk_desc2.getShape(),
+                                                                                     blk_desc2.getBlockDims(), blk_desc2.getOrder(),
+                                                                                     blk_desc2.getOffsetPadding(), blk_desc2.getOffsetPaddingToData(),
+                                                                                     blk_desc2.getStrides());
+
+        return ((*cpu_blk_desc_ptr_ncsp).isCompatible(*cpu_blk_desc_ptr_nspc));
+    };
 
-TEST(MemDescTest, CheckScalar) {
-    GTEST_SKIP();
+    ASSERT_FALSE(isCompatibleEquallyStoredInMemory(dnnl::memory::format_tag::nchw, dnnl::memory::format_tag::nhwc,
+                                                   Shape(SizeVector{1, 3, 1, 1})));
+    ASSERT_FALSE(isCompatibleEquallyStoredInMemory(dnnl::memory::format_tag::nchw, dnnl::memory::format_tag::nhwc,
+                                                   Shape(SizeVector{1, 1, 2, 3})));
+    ASSERT_FALSE(isCompatibleEquallyStoredInMemory(dnnl::memory::format_tag::nchw, dnnl::memory::format_tag::nhwc,
+                                                   Shape(ngraph::PartialShape{{1}, {1, 3}, {1, 2}, {1, 3}})));
+    ASSERT_FALSE(isCompatibleEquallyStoredInMemory(dnnl::memory::format_tag::abc, dnnl::memory::format_tag::bac,
+                                                   Shape(ngraph::PartialShape{{1}, {2, 3}, {4, 5}})));
 }
 
-TEST(MemDescTest, UpperBound) {
-    GTEST_SKIP();
-}
+TEST(MemDescTest, isCompatibleFalseTests) {
```

Comment:
please separate test cases for `DnnlMemoryDesc` `DnnlBlockedMemoryDesc` `CpuBlockedMemoryDesc`
as you did in `isCompatibleTrueTests`